ci-operator/config/maistra/istio/maistra-istio-maistra-3.0.yaml

```diff
@@ -69,6 +69,64 @@ tests:
           memory: 100Mi
       timeout: 2h0m0s
     workflow: servicemesh-istio-e2e-aws
+- as: maistra-istio-unit-3-0
+  commands: |
+    make -e T=-v build racetest binaries-test \
```

Comment:
This failed with the git error:
```
fatal: detected dubious ownership in repository at '/go/src/github.com/maistra/istio'
```

This is because the upstream image uses an old `git`. We should add the workaround we had before: https://github.com/openshift/release/pull/35735/files - But only for these 3.0 jobs.